dts/arm/adi/max32/max32655.dtsi

```diff
@@ -30,6 +30,34 @@
 	};
 };
 
+&timer0 {
+	pwm {
+		pinctrl-0 = <&tmr0a_ioa_p0_2>;
+		pinctrl-names = "default";
+	};
+};
+
+&timer1 {
+	pwm {
+		pinctrl-0 = <&tmr1a_ioa_p0_14>;
+		pinctrl-names = "default";
+	};
+};
+
+&timer2 {
+	pwm {
+		pinctrl-0 = <&tmr2a_ioa_p0_26>;
+		pinctrl-names = "default";
+	};
+};
+
+&timer3 {
+	pwm {
+		pinctrl-0 = <&tmr3a_ioa_p1_6>;
+		pinctrl-names = "default";
+	};
+};
+
```

Comment:
There are multiple pins can be used for timer output, so that pin related default value definition shall be defined in board.dtsi file depend on the board schematic.

Reply:
Done